src/NUnitEngine/nunit.engine.netstandard/nunit.engine.netstandard.csproj

```diff
@@ -12,6 +12,11 @@
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <PackageId>nunit.engine.netstandard</PackageId>
+    <Title>NUnit .NET Standard Engine</Title>
+    <Authors>Charlie Poole</Authors>
+    <Description>Provides a common interface for loading, exploring and running NUnit tests in .NET Core and .NET Standard</Description>
```

Comment:
"Provides a common interface" makes me think this would be the nunit.engine.api assembly (which we also have a NuGet package for) - but this is the full engine, isn't it? Could the description be reworded - or possibly the same as the main package, with .NET Core and .NET Standard named, of course.